src/rt/tracegc.d

```diff
@@ -88,243 +84,156 @@ extern (C) void[] _d_newarrayT(const TypeInfo ti, size_t length);
 extern (C) void[] _d_newarrayiT(const TypeInfo ti, size_t length);
 extern (C) void[] _d_newarraymTX(const TypeInfo ti, size_t[] dims);
 extern (C) void[] _d_newarraymiTX(const TypeInfo ti, size_t[] dims);
-extern (C) void* _d_newitemT(in TypeInfo _ti);
-extern (C) void* _d_newitemiT(in TypeInfo _ti);
-
-private string generatePrintf ( )
-{
-    version (tracegc)
-    {
-        return q{
-            printf("%s file = '%.*s' line = %d function = '%.*s'\n",
-                __FUNCTION__.ptr,
-                file.length, file.ptr,
-                line,
-                funcname.length, funcname.ptr
-                );
-        };
-    }
-    else
-        return "";
-}
```

Comment:
Funnily, I did exactly to simplify reviewing, minimizing amount of lines removed in the commit that does interesting stuff (as tracing copy-paste bits have been removed in previous commit) :)